src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const BIGGER_DISCONT = 7;
+  const SMALLER_DISCONT = 3;
+  let sum = days * PAYMENT;
+
+  if (days >= BIGGER_DISCONT) {
+    sum -= 50;
+
+    return sum;
+  }
+
+  if (days >= SMALLER_DISCONT) {
+    sum -= 20;
+
+    return sum;
```

Comment:
Similar to the previous comment, the blank line after this 'return' statement is unnecessary and should be removed.

Reply:
when adding requesting changes can't pass linter